### PR TITLE
more consistency for master

### PR DESCRIPTION
**Pull request by OskarStark**
Its much easier to review commit by commit

Todo before merging this PR:
- [x] Merge https://github.com/symfony/symfony-docs/pull/11243
- [x] Upmerge to `master`
- [x] Rebase this PR

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
Nice!

**Comment by OskarStark** [comment]
> Nice!

Spotted by "the tool" 😃

**Comment by OskarStark** [comment]
Please wait while merging this! I need to fix sth. else

**Comment by javiereguiluz** [comment]
Don't worry, I'm not going to merge this. I'm going to leave it to you, so you can handle it as soon as you are granted the merge permission 🎉

**Comment by OskarStark** [comment]
Todo list complete, please review again, thanks!

File: reference/forms/types/options/attr_translation_parameters.rst.inc
Comment (xabbuh) [inline reference/forms/types/options/attr_translation_parameters.rst.inc]: Add a traling comma here too?

File: reference/forms/types/options/help_translation_parameters.rst.inc
Comment (xabbuh) [inline reference/forms/types/options/help_translation_parameters.rst.inc]: same here

File: reference/forms/types/options/label_translation_parameters.rst.inc
Comment (xabbuh) [inline reference/forms/types/options/label_translation_parameters.rst.inc]: and here

**Review by xabbuh — APPROVED** [review]

**Pull request merged**
